README.md

```diff
@@ -157,6 +160,24 @@ Echo.join('channel3');
 Echo.leaveChannel("presence-channel3")
 ```
 
+**2. Error handling**
+- [Ably echo client](https://github.com/ably-forks/laravel-echo) emit [ably specific error codes](https://github.com/ably/ably-common/blob/main/protocol/errors.json) instead of [pusher error codes](https://pusher.com/docs/channels/library_auth_reference/pusher-websockets-protocol/#error-codes).
+- Aim is to make sure error details are descriptive and easy to understand, so it's more effective to take a corrective action ( Pusher errors lack error context, mostly emitted as integer error codes ).
```

Comment:
I think remove this line, it seems a bit dismissive of Pusher SDKs and doesn't add much value

Reply:
I think it should be dismissive. 
I would say error details add much more value rather than just emitting int error code. 
Pusher SDKs truly suck at handling errors. It's hard to find out what's going wrong. This was one of the reasons why lot of people used to ask questions about `ably-pusher` setup, and it does take a considerable amount of time to fix it. I personally faced tons of issues while the initial setup.
I can confidently say, this is true for all pusher SDK users.

Reply:
Okay I have updated note to =>
```
Ably emitted errors are descriptive and easy to understand, so it's more effective to take a corrective action ( Pusher errors mostly emitted as integer error codes ).
```
Now, it's not dismissive and mentions that ably errors are more informative.

Reply:
That's better but I would still prefer to remove the "( Pusher errors mostly emitted as integer error codes )" bit tbh

Reply:
done